pkg/apis/servicecatalog/v1beta1/class.go

```diff
@@ -81,6 +81,16 @@ func (c *ClusterServiceClass) GetServiceBrokerName() string {
 	return c.Spec.ClusterServiceBrokerName
 }
 
+// GetIsManaged returns whether the class is managed or not.
+func (c *ServiceClass) GetIsManaged() bool {
```

Comment:
Why not `IsManaged()` ?

Reply:
Thanks! IsManaged() is much better, I will change this as you suggested. 

Currently I have removed the new commit from this PR  because it fixes another issue. I will submit the commit as a new PR once this PR is merged.

@luksa Do you know when this PR can be merged?